cicdscripts/PublishPackage.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [string[]]
+    $ReleaseBranches
+)
+
+$ErrorActionPreference = "Stop"
+
+$regexBranches = @()
+foreach ($branch in $ReleaseBranches -split ",")
+{
+	$regexBranches +=  "(^" + $branch + "*)"
+}
+$regex = $regexBranches -join "|"
+
+Write-Host $env:BRANCH_NAME
+Write-Host $regex
+
+<#
+Regex matching release branches
+#>
+$reBranch = [regex]$regex
+
+if ($env:BRANCH_NAME -match $reBranch) 
+{
+	try {
+
+		#deploy already built package.
+		$assemblyPath = "src\GregClient\bin\release"
+		$nupkgFile = Get-ChildItem $assemblyPath\*.nupkg -Depth 1
```

Comment:
How is this nupkg file being generated?

Reply:
https://github.com/DynamoDS/PackageManagerClient/blob/master/src/GregClient/GregClient.csproj#L20
using this csproj property